### ISSUE TITLE
Consider an OpenCollective for development rather than a Patreon

### ISSUE THREAD

**Issue by BobbyMcWho** [rails-sqlserver/activerecord-sqlserver-adapter#654]
An increasing number of projects (including Babel, Vue, RubyTogether) are securing funding for maintenance via https://opencollective.com/discover I think this is a nice and transparent way to solicit donations to the project, with minimal maintenance needs for the repository owner, and nice recognition levels for the backers themselves.

**Comment by caseyprovost** [comment]
Ideally, I think the project would benefit most from a company's backing. Maybe Microsoft itself would be willing to donate some dev hours to this. Assuming this library is a critical piece in some Ruby apps that are making businesses money...they might want the opportunity to invest here as well.

**Comment by jayjlawrence** [comment]
I was attempting to support via Patreon. I checked the link - the profile has been removed yet Patreon has continued to grab my cash. I have no idea what happened to the funds that I've contributed. Obviously I am asking for a refund.  

I'm really disappointed here - the "README.me" was updated 2 months ago and still references this "Its ongoing development is made possible thanks to the support by these awesome backers. If you'd like to join them, check out our Patreon Campaign." To a list of "this could be " blank entries and a now non-existent Patreon campaign.
